Plotter/config/samples.py

```diff
@@ -154,12 +155,15 @@ def getsampleset(channel,era,**kwargs):
     LOG.throw(IOError,"Did not recognize era %r!"%(era))
   
   # OBSERVED DATA SAMPLES
-  if   'tautau' in channel: dataset = "Tau_Run%d?"%year
-  elif 'mutau'  in channel: dataset = "SingleMuon_Run%d?"%year
-  elif 'etau'   in channel: dataset = "EGamma_Run%d?"%year if year==2018 else "SingleElectron_Run%d?"%year
-  elif 'mumu'   in channel: dataset = "SingleMuon_Run%d?"%year
-  elif 'emu'    in channel: dataset = "SingleMuon_Run%d?"%year
-  elif 'ee'     in channel: dataset = "EGamma_Run%d?"%year if year==2018 else "SingleElectron_Run%d?"%year
+  if   'tautau'   in channel: dataset = "Tau_Run%d?"%year
+  elif 'mutau'    in channel: dataset = "SingleMuon_Run%d?"%year
+  elif 'etau'     in channel: dataset = "EGamma_Run%d?"%year if year==2018 else "SingleElectron_Run%d?"%year
+  elif 'mumu'     in channel: dataset = "SingleMuon_Run%d?"%year
+  elif 'emu'      in channel: dataset = "SingleMuon_Run%d?"%year
+  elif 'ee'       in channel: dataset = "EGamma_Run%d?"%year if year==2018 else "SingleElectron_Run%d?"%year
+  elif 'mumutau'  in channel: dataset = "SingleMuon_Run%d?"%year
+  elif 'eetau'    in channel: dataset = "EGamma_Run%d?"%year if year==2018 else "SingleElectron_Run%d?"%year  
```

Comment:
These two lines
```
elif 'mumutau' in channel: ...
elif 'eetau' in channel: ...
```
seem redundant, because of
```
elif 'mutau' in channel: ...
elif 'etau' in channel: ...
```
few lines above